modules/api/src/main/java/net/countercraft/movecraft/util/Tags.java

```diff
@@ -71,4 +71,16 @@ public static NamespacedKey keyFromString(@NotNull String string){
             return null;
         }
     }
+
+    @NotNull
+    public static EnumSet<Material> getMaterialsFromString(@NotNull String materialName) {
```

Comment:
Since this is api, include javadoc

Reply:
perhaps opt for an alternative method name, such as `parseMaterials` to indicate that this is not a simple conversion. This also removes the redundant `FromString` suffix.